README.md

```diff
@@ -136,7 +136,7 @@ For Artifactory installation, you can download the `.env` file [here](https://ra
 
 Create a secret for JFrog's admkin token - [Access Token](https://jfrog.com/help/r/how-to-generate-an-access-token-video/artifactory-creating-access-tokens-in-artifactory) using any of the following methods
 ```shell
-kubectl create secret generic jfrog-admin-token --from-file=<path_to_license_file>token
+kubectl create secret generic jfrog-admin-token --from-file=<path_to_token_file>token
```

Comment:
Do we need the last `token` part? I thought just the path to the file is enough for the `--from-file` arg?

Reply:
I agree with @alexhung . If we don't need it, let's remove that.

Reply:
@alexhung @mritunjaykumar The reason for adding _token_ is that it is being used as a key for the secret at https://github.com/jfrog/log-analytics-datadog/pull/41/files#diff-52dafdaea0db08888a54e98fa399f8cb088a5498f38f992919467c8f2d79b6ffR34

Reply:
That's not how the key should be defined according to the doc: https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/#use-source-files

For this case, it would be:

```sh
kubectl create secret generic jfrog-admin-token --from-file=token=<path_to_token_file>
```

Reply:
Also, I don't think the `key` is necessary since it's a single item secret.